varnish-sys/src/vcl/ws.rs

```diff
@@ -45,29 +46,38 @@ impl<'a> Workspace<'a> {
     }
 
     /// Allocate a `[u8; sz]` and return a reference to it.
+    ///
+    /// # Safety
+    /// Allocated memory is not initialized with zeroes
```

Comment:
So we want a safe version that memsets the allocated memory?

Reply:
no, zero-ed memory is not the same - if you have a struct with a constructor that sets one of the fields to a magic number, creating that struct with zeroes is still incorrect.  Instead, we could introduce one or more functions in Workspace like thees:

```rust
fn alloc_default<T: Default>(&mut self) -> &mut T { ... }
fn alloc_from<T>(&mut self, instance: T) -> &mut T { ... }
```